bindings/python/src/modelling/expose-pinocchio-functions.cpp

```diff
@@ -52,6 +53,11 @@ void exposeFrameFunctions() {
   using FrameTranslation = FrameTranslationResidualTpl<Scalar>;
   using FrameTranslationData = FrameTranslationDataTpl<Scalar>;
 
+  using FrameCollision = FrameCollisionResidualTpl<Scalar>;
+  using FrameCollisionData = FrameCollisionDataTpl<Scalar>;
+
+  using GeometryModel = pinocchio::GeometryModel;
```

Comment:
_Coding-style point:_ You can use a using-decl instead of the using-typedef:
```suggestion
  using pinocchio::GeometryModel;
```